Zend/zend_operators.h

```diff
@@ -490,6 +490,20 @@ overflow: ZEND_ATTRIBUTE_COLD_LABEL
 	return;
 overflow: ZEND_ATTRIBUTE_COLD_LABEL
 	ZVAL_DOUBLE(op1, (double)ZEND_LONG_MAX + 1.0);
+#elif defined(HAVE_ASM_GOTO) && defined(__aarch64__)
+	register long tmp asm("x5");
+	__asm__ goto (
+		"ldr %0, [%1]\n\t"
+		"adds %0,%0,1\n\t"
+		"bvs %l2\n"
+		"str %0, [%1]"
+		:
+		: "r"(tmp), "r"(&op1->value)
```

Comment:
Is `"r"(tmp)` really right here? This register is being written to.

Reply:
You are right. However the `asm goto` statement does not allow registers to be written to, so I cannot declare `tmp` as read+write: `"+r"(tmp)`.